Unreachable fsync call in SysCommandWorker.write

### Description

`pylint --enable=unreachable .` picks up the following unreachable `fsync` call:

https://github.com/archlinux/archinstall/blob/7437668f1e51eb3c88f7a0ddc7ad7ef30b7f64a5/archinstall/lib/general.py#L217-L219

### Warning
```
************* Module archinstall.lib.general
archinstall/lib/general.py:219:3: W0101: Unreachable code (unreachable)
```

--- a/archinstall/lib/general.py
+++ b/archinstall/lib/general.py
@@ -220,7 +220,6 @@ class SysCommandWorker:
 
 		if self.child_fd:
 			return os.write(self.child_fd, data + (b'\n' if line_ending else b''))
-			os.fsync(self.child_fd)
 
 		return 0
 

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -160,7 +160,6 @@ disable = [
     "fixme",
     "protected-access",
     "raise-missing-from",
-    "unreachable",
     "unspecified-encoding",
     "unused-argument",
 ]

